pandas/tests/io/formats/test_format.py

```diff
@@ -2402,8 +2402,8 @@ def test_east_asian_unicode_series(self):
 
                 s.index = ["ああ", "いいいい", "う", "えええ"]
                 expected = (
-                    "ああ            あ\n"
-                    "            ...   \n"
+                    "ああ            あ\n            "
+                    "...   \n"
```

Comment:
I think in this case it's clearer the other way round. I know we can't have exceptions if we want to validate it in the CI, but I'm wondering if we could just apply the rule for a single space, and leave those other cases to common sense.

Reply:
> I know we can't have exceptions if we want to validate it in the CI

Actually I have thought of a way to do so (*ish*).

currently the script is validating if there is no space at the end of the line and a leading space at the begging of the next one, but what if it having an exception to check the number of leading spaces and according to that, the script will know if this is a style violation or an intentional.

If we say for example that if the string is starting with ```3``` spaces, it will consider it as intentional indentation, and will not report errors.

I am not very good with words so here is some test cases.

---

## Will __not__ pass:

```python
foo = (
            "bar"
             " baz" # One space
)
```

```python
foo = (
            "bar"
             "  baz" # Two spaces
)
```

---

## Will pass:

```python
foo = (
            "bar"
             "   baz" # Three spaces
)
```

---

Any thoughts?

Reply:
I see your point, but doing something so complex is out of scope for pandas. I'd rather don't validate this, that making contributors have to know of such a thing.

I'd simply check the case where there are no spaces at the end of the first line, and there is just one space in the next. This will capture written paragraphs, where we should never have one space next to another, and will ignore everything else, which may include csv data and other stuff.